tubular/tests/test_panel_backend.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+
+import mock
+
+from tubular.panel_backend_api import PanelBackendAPI
+
+class PanelBackendAPITests(TestCase):
+    """
+    Unit tests for Panel Backend.
+    """
+
+    def setUp(self):
+        """
+        Setup data for test cases.
+        """
+        self.base_url = 'https://example.com'
+        self.auth_token = 'token'
+
+
```

Comment:
nit: there should be only one new line between methods

Reply:
done